src/content/en/updates/2017/01/nic56.md

```diff
@@ -0,0 +1,140 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: What's new in Chrome 56 for developers? Web Bluetooth API, CSS Position Sticky, HTML5 by Default is enabled for all users.
+
+{# wf_published_on: 2017-01-25 #}
+{# wf_updated_on: 2017-01-25 #}
+{# wf_featured_image: /web/updates/images/generic/new-in-chrome.png #}
+{# wf_tags: chrome55,new-in-chrome,css,layout,html5,flash,webbluetooth,webvr #}
+{# wf_featured_snippet: With Chrome 56, web apps can now communicate with nearby Bluetooth Low Energy devices using the <b>Web Bluetooth API</b>, CSS <code>position: sticky;</code> is back - making it easy to create elements that scroll normally until sticking to the top of the viewport. And HTML5 by Default is enabled for all users. #}
+
+# New In Chrome 56 {: .page-title }
+
+{% include "web/_shared/contributors/petelepage.html" %}
+
+<div class="clearfix"></div>
+
+<div class="video-wrapper">
+  <iframe class="devsite-embedded-youtube-video" data-video-id="F4DfGVbvRpY"
+          data-autohide="1" data-showinfo="0" frameborder="0" allowfullscreen>
+  </iframe>
+</div>
+
+* Web apps can now communicate with nearby Bluetooth Low Energy
+  devices using the [Web Bluetooth API](#webbluetooth).
+* [`position: sticky`](#position-sticky) is back - making it easy to create
+  elements that scroll normally until sticking to the top of the viewport.
+* And [HTML5 by Default](#html5-by-default) is enabled for all users.
+
+I’m Pete LePage, let’s dive in and see what’s new for developers in Chrome 56! 
+
+<div class="clearfix"></div>
+
+## Web Bluetooth API {: #webbluetooth }
+
+Until now, the ability to interact with bluetooth devices has only been
+possible for native apps.
+
+With Chrome 56, your web apps can communicate with nearby Bluetooth Low Energy
+devices in a secure and private manner using the
+[Web Bluetooth API](//webbluetoothcg.github.io/web-bluetooth/). 
+
+The Web Bluetooth API uses the [GATT protocol](https://webbluetoothcg.github.io/web-bluetooth/),
+which enables your app to connect to devices such as light bulbs, toys,
+heart-rate monitors, LED displays and more with just a
+[few lines of JavaScript](https://googlechrome.github.io/samples/web-bluetooth/).
+Web Bluetooth can also be combined with Physical Web beacons to discover and control nearby devices.
+
+Francois has a great
+[article on Updates](/web/updates/2015/07/interact-with-ble-devices-on-the-web),
```

Comment:
Is this a word-for-word transcript? Is it meant to be? 'John has an article' is probably fine for a video. 'Read John's article' is stronger and more appropriate for an article. Your call. If you change it, do so throughout.